pkg/lwip/contrib/netdev/lwip_netdev.c

```diff
@@ -73,15 +74,19 @@ static err_t slip_output6(struct netif *netif, struct pbuf *q, const ip6_addr_t
 #endif
 static void _event_cb(netdev_t *dev, netdev_event_t event);
 static void *_event_loop(void *arg);
+static void _isr(event_t *ev);
 
 err_t lwip_netdev_init(struct netif *netif)
 {
     LWIP_ASSERT("netif != NULL", (netif != NULL));
     LWIP_ASSERT("netif->state != NULL", (netif->state != NULL));
+    lwip_netif_t *compat_netif = container_of(netif, lwip_netif_t, lwip_netif);
+    compat_netif->ev_isr.handler = _isr;
     netdev_t *netdev;
     netopt_enable_t enabled = 0;
     uint16_t dev_type;
     err_t res = ERR_OK;
+    event_queue_init_detached(&lwip_event_queue);
```

Comment:
It seems this assumes there is only a single netdev? Each should have their own event queue?

Reply:
Hm, why? We don't have SMP, so events are processed sequentially anyway - a single event queue for all netdevs should be fine.

Reply:
This will zero the struct, and it could have something in it by the time this is called a second time I think

Reply:
Good catch!
This should better be placed in `lwip_bootstrap` then.

Reply:
With the new initialization this line should now be removed

Reply:
Indeed, thx. Fixed :)